src/main/java/seedu/findvisor/model/person/Meeting.java

```diff
@@ -56,6 +55,18 @@ public static boolean isValidRemark(String remark) {
         return remark.length() <= MAX_REMARK_LENGTH;
     }
 
+    public LocalDateTime getStart() {
+        return start;
+    }
+
+    public LocalDateTime getEnd() {
+        return end;
+    }
+
+    public String getRemark() {
+        return remark;
+    }
+
```

Comment:
Just wanted to clarify for the implementation of these getter methods are to enhance the abstraction for retrieving the `Meeting` fields or from a more OOP-focused perspective? Perhaps would be better to set the `Meeting` fields to private since the getter methods are present?

Reply:
Actually added these when I was doing some functional stuff with `Optionals`, so I can do things like `Meeting::getRemark` but didn't need it in the end. I'll refactor the fields to private, to standardize the way to access.

Reply:
Actually I'll do it in a seperate PR to not further increase the size of this PR